src/components/Explore/TokenRow.tsx

```diff
@@ -0,0 +1,279 @@
+`
+
+const MarketCapCell = styled(Cell)`
+  justify-content: flex-end;
+  padding: 12px 8px;
+  gap: 10px;
+  min-width: max-content;
+  ${({ theme }) => theme.mediaWidth.upToSmall`
+  display: none;
+`};
+`
+const VolumeCell = styled(Cell)`
+  justify-content: flex-end;
+  padding: 12px 8px;
+  gap: 10px;
+  ${({ theme }) => theme.mediaWidth.upToMedium`
+    display: none;
+  `};
+`
+const SparkLineCell = styled(Cell)`
+  flex-direction: column;
+  padding: 16px 24px;
+  gap: 10px;
+  min-width: 120px;
+  ${({ theme }) => theme.mediaWidth.upToLarge`
+    display: none;
+  `};
+`
+const SparkLineImg = styled(Cell)`
+  max-width: 124px;
+  max-height: 28px;
+  flex-direction: column;
+  transform: scale(1.2);
+`
+
+const SwapCell = styled(Cell)`
+  flex-direction: column;
+  padding: 16px 8px;
+  gap: 10px;
+  ${({ theme }) => theme.mediaWidth.upToExtraSmall`
+  display:none
+`};
+`
+
+const SwapButton = styled.button`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  padding: 8px;
+  gap: 6px;
+  font-weight: 600;
+
+  width: 54px;
+  height: 32px;
+
+  background: ${({ theme }) => theme.primary2};
+  border-radius: 12px;
+  border: none;
+  color: ${({ theme }) => theme.white};
+  &:hover {
+    background-color: ${({ theme }) => darken(0.05, theme.primary2)};
+  }
+`
+
+const TokenSymbol = styled.span`
+  color: ${({ theme }) => theme.text3};
+`
+const ArrowCell = styled.div`
+  padding-left: 4px;
+  display: flex;
+  flex-direction: column;
+`
+
+const SortingCategory = styled.span`
+  color: ${({ theme }) => theme.primary1};
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`
+
+const SortArrowCell = styled(Cell)`
+  flex-direction: column;
+  padding-right: 2px;
+`
+
+function getHeaderCategory(category: string, sortCategory: string, sortDecreasing: boolean) {
+  if (sortCategory === category) {
+    return (
+      <SortingCategory>
+        <SortArrowCell>{sortDecreasing ? <ArrowDown size={14} /> : <ArrowUp size={14} />}</SortArrowCell>
+        {category}
+      </SortingCategory>
+    )
+  }
+  return category
+}
+
+export function HeaderRow() {
+  /* TODO: access which sort category used and timeframe used (temporarily hardcoded values) */
+  const sortCategory = SORT_CATEGORIES[1]
+  const sortDecreasing = true
+  return (
+    <HeaderRowWrapper>
+      {/* Empty contents for no header for favorite and rank columns */}
+      <FavoriteCell></FavoriteCell>
+      <ListNumberCell></ListNumberCell>
+      <NameCell>
+        <Trans>Name</Trans>
+      </NameCell>
+      <PriceCell>
+        <Trans>{getHeaderCategory('Price', sortCategory, sortDecreasing)}</Trans>
+      </PriceCell>
+      <PercentChangeCell>
+        <Trans>{getHeaderCategory('% Change', sortCategory, sortDecreasing)}</Trans>
+      </PercentChangeCell>
+      <MarketCapCell className="col-hide-3">
+        <Trans>{getHeaderCategory('Market Cap', sortCategory, sortDecreasing)}</Trans>
```

Comment:
discussed in slack, but you can make a component called `HeaderCellContent` which takes `isSorted`, `sortDecreasing`, `sortable`, etc.